Update sample to use the new URL field (#1121)
knative/docs: docs/serving/samples/blue-green-deployment.md

(http://blue-green-demo.default.YOUR_CUSTOM_DOMAIN.com) a few times to verify
that no traffic is being routed to v1 of the app.

We added a named route to v1 of the app, so you can now access it at the URL

> listed in the traffic block of status. To get the url you can execute the

Reviewer: I would say something like "..listed in the traffic block of the status section" or "list in the status section of the route manifest". 

"traffic block of status" does not seem right.

> listed in the traffic block of status. To get the url you can execute the

Reviewer 2: ```suggestion
listed in the traffic block of the status section. To get the URL, enter the
```

Reviewer 2: Per @sebgoa 's suggestion

> command below:

Reviewer 2: ```suggestion
following command:
```
